### PR TITLE
Leaderboard Final Touches

### PR DESCRIPTION
**Pull request by JEFFKAY16**
## Leaderboard Final Touches

In this updates, I:
- Added styling as a way to improve the feel and appearance of the app while maintaining the structure.
- Made the app responsive

### CODE REVIEW

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @JEFFKAY16, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Good job with UI
:heavy_check_mark: Your app is running

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

Please also consider:
- [x] As it is a requirement for this project, please in your readme file, `Getting Started` section, please kindly add instructions to install dependencies(e.g. `npm i`) and to start the app(e.g. `npm start`). 



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,50 +1,111 @@
...
 Contributions, issues, and feature requests are welcome!
-please reach me on my [Twitter Account](https://twitter.com/jeff_kayombo)
+
+Feel free to check the [issues page](../../issues/).
 
 ## Show your support
 
-Give a ⭐️ if you like this project!
+Please 🙏 Please 🙏... Give a ⭐️ if you like this project!
+
```
Comment (carlos-ssh) [inline README.md:78]: # 💾 💾 💽 [OPTIONAL]   💽 💾 💾
- Link a license to your ``README`` file.

File: src/index.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
   <section class="container">
     <h1>Leaderboard</h1>
     <div class="d-flex">
-      <div class="recent-scrore">
+      <div class="recent-score">
```
Comment (carlos-ssh) [inline src/index.html:16]: - Kindly fix this issue related to your ``POST`` displayed in console:
Screen Shot 2021-11-20 at 6 24 17

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @JEFFKAY16 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 2 above)
Comment (JEFFKAY16) [reply src/index.html:16]: @carlos-ssh 

Thanks for the review, I noticed that when I run the project on my computer, everything works perfectly without a single error in the console. I also took note that the URL displaying in your browser is `localhost:8080` while mine is being run on `localhost:8083`, which could potentially mean you're not running the correct project. Please make sure that they are no other projects running on your computer. I'll wait for your feedback, please
Screenshot 2021-11-20 at 2 40 32 PM

Screenshot 2021-11-20 at 2 52 09 PM

Comment (JEFFKAY16) [reply src/index.html:16]: > * Kindly fix this issue related to your `POST` displayed in console:
>   Screen Shot 2021-11-20 at 6 24 17

Hi @carlos-ssh ,

This error was a result of your internet connection failing to resolve the API endpoint.

Have you considered running the project again, with a stable connection?

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @JEFFKAY16, 

Everything is working fine for me. Well Done.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mimipeshy** [comment]
## Hello @JEFFKAY16  👋,

_This is an additional review as a result of the second opinion request._

Apologies for the late reply, this won't happen again. However congratulation on fixing the issues. 😃 

Keep rocking 🚀!

**Pull request merged**